docs/root/_include/tcp_stats.rst

```diff
@@ -14,8 +14,22 @@
    cx_tx_retransmitted_segments, Counter, Total TCP segments retransmitted
    cx_rx_bytes_received, Counter, Total payload bytes received for which TCP acknowledgments have been sent.
    cx_tx_bytes_sent, Counter, Total payload bytes transmitted (including retransmitted bytes).
+   cx_data_segments_delivered, Counter, Data segments delivered to the receiver including retransmits
+   cx_lost, Counter, Lost TCP segments
    cx_tx_unsent_bytes, Gauge, Bytes which Envoy has sent to the operating system which have not yet been sent
    cx_tx_unacked_segments, Gauge, Segments which have been transmitted that have not yet been acknowledged
    cx_tx_percent_retransmitted_segments, Histogram, Percent of segments on a connection which were retransmistted
    cx_rtt_us, Histogram, Smoothed round trip time estimate in microseconds
    cx_rtt_variance_us, Histogram, Estimated variance in microseconds of the round trip time. Higher values indicated more variability.
+   cx_rcv_rtt, Histogram, Receiver side round trip time estimate in microseconds
+   cx_tx_window_scale, Histogram, Shift value to scale send window
+   cx_rx_window_scale, Histogram, Shift value to scale receive window
+   cx_reordering, Histogram, Maximum observed reordering distance
+   cx_rto, Histogram, Retransmission timeout in microseconds
```

Comment:
Suffix with `_us` for this and others that have a known unit

Reply:
I tried to suffix it with `_us` but got formatting errors `Don't suffix histogram names with the unit symbol` so removed it.